cmd/lakefs/cmd/config.go

```diff
@@ -0,0 +1,220 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/manifoldco/promptui"
+	"github.com/mitchellh/go-homedir"
+
+	"io/ioutil"
+
+	"encoding/json"
+
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+)
+
+// configuration model
+
+type Database struct {
+	ConnectionString string `yaml:"connection_string"`
+}
+
+type Gateways struct {
```

Comment:
Instead of using dedicated structs, I would use a map.

Reply:
So can I use only one structure `Configuration` and for rest I can use map, or should I use map for all of em?